erpnext/portal/product_configurator/test_product_configurator.py

```diff
@@ -52,9 +56,9 @@ def test_get_products_for_website(self):
 
 
 	def create_variant_item(self):
-		if not frappe.db.exists('Item', '_Test Variant Item 1'):
+		if not frappe.db.exists('Item', 'Test Variant PRT'):
```

Comment:
```suggestion
		if not frappe.db.exists('Item', '_Test Variant Item 1'):
```
why are we checking if a different item exists and creating a different item?